golos.vesting/golos.vesting.cpp

```diff
@@ -326,7 +323,7 @@ void vesting::create(symbol symbol, name notify_acc) {
     });
 }
 
-void vesting::calculate_convert_vesting() {
+void vesting::calculate_convert_vesting(uint64_t, symbol symbol) {
```

Comment:
there is no need to pass symbol here, coz following `for` process all vesting tokens:
https://github.com/GolosChain/golos-smart/blob/5882b2ecd6fc861ef511df61a1905813e89e6bf4/golos.vesting/golos.vesting.cpp#L334-L335
get params symbol same way as done for `convert_table`